### PR TITLE
build: hybridsql docker 0.5.0

### PR DESCRIPTION
**Pull request by aceforeverd**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

    update compile image:
    - upgrae thirdparty to 0.5.0, bundled rocksdb

### CODE REVIEW

**Comment by aceforeverd** [comment]
block until #1482 is resolved

File: Makefile [hunk 1]
```diff
@@ -142,7 +142,6 @@ thirdparty-fast:
 		echo "[deps]: thirdparty up-to-date. reinstall zetasql from $(ZETASQL_VERSION) to $$new_zetasql_version"; \
 		$(MAKE) thirdparty-configure; \
 		$(CMAKE_PRG) --build $(THIRD_PARTY_BUILD_DIR) --target zetasql; \
-		# FIXME(zhanghao): remove this rocksdb fix after we update thirdparty \
 		$(CMAKE_PRG) --build $(THIRD_PARTY_BUILD_DIR) --target rocksdb; \
```
Comment (zhanghaohit) [inline Makefile]: the rocksdb target can be removed

File: Makefile [hunk 2]
```diff
@@ -151,7 +150,6 @@ thirdparty-fast:
 	    echo "[deps]: install zetasql only"; \
 	    $(MAKE) thirdparty-configure; \
 	    $(CMAKE_PRG) --build $(THIRD_PARTY_BUILD_DIR) --target zetasql; \
-	    # FIXME(zhanghao): remove this rocksdb fix after we update thirdparty \
 	    $(CMAKE_PRG) --build $(THIRD_PARTY_BUILD_DIR) --target rocksdb; \
```
Comment (zhanghaohit) [inline Makefile]: same here

**Comment by aceforeverd** [comment]
[Compile & Docker Image](https://trello.com/c/OkCVyzne/43-compile-docker-image)

**Review by vagetablechicken — APPROVED** [review]

**Review by zhanghaohit — APPROVED** [review]

**Pull request merged**